src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -189,11 +189,14 @@ private IEnumerable<NuGet.Credentials.ICredentialProvider> GetCredentialProvider
             var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings, Console)
                 .BuildAll(Verbosity.ToString())
                 .ToList();
+            var securePluginProviders =  (new SecureCredentialProviderBuilder(PluginManager.Instance, Console)).BuildAll().Result;
```

Comment:
use `await` instead of `Result` since `Result` has potential for deadlock if Task has not completed yet.